app/api/api/views.py

```diff
@@ -740,6 +761,97 @@ def create(self, request, *args, **kwargs):
         )
 
 
+class ScanReportConceptViewSetV2(viewsets.ModelViewSet):
+    """
+    Version V2.
+    """
+
+    queryset = ScanReportConcept.objects.all()
+    serializer_class = ScanReportConceptSerializer
+
+    def create(self, request, *args, **kwargs):
+        body = request.data
+
+        # Extract the content_type
+        content_type_str = body.pop("content_type", None)
+        content_type = ContentType.objects.get(model=content_type_str)
+        body["content_type"] = content_type.id
+
+        # validate person_id and date event are set on table
+        table_id = body.pop("table_id", None)
+        try:
+            table = ScanReportTable.objects.get(pk=table_id)
+        except ObjectDoesNotExist:
+            return Response(
+                {"detail": "Table with the provided ID does not exist."},
+                status=400,
```

Comment:
```suggestion
                status=status.HTTP_404_NOT_FOUND,
```
404 not found is more appropriate here if a resource doesn't exist